requirements.txt

```diff
@@ -1,4 +1,4 @@
-fsspec>=2021.4.0
+fsspec
 intake
-s3fs
-msgpack
+requests
```

Comment:
Any idea why we need this, when we don't use it explicitly in the codebase?

Reply:
https://github.com/intake/intake/issues/706
if Martin does another intake release we can remove

Reply:
🦸